dcos/nodeutil/helper.go

```diff
@@ -0,0 +1,8 @@
+package nodeutil
+
+import "regexp"
+
+func validateUUID(text string) bool {
+	r := regexp.MustCompile("[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[8|9|aA|bB][a-f0-9]{3}-[a-f0-9]{12}")
```

Comment:
I only mentioned lower/upper case because your regexp uses a mix of constraints, some asking for lower case, some asking for either